README.md

```diff
@@ -1 +1,14 @@
-# django-imager
\ No newline at end of file
+# imagersite
+
+**Authors**: David Snowberger, Keith Eckhert
+
+**Version**: 0.1.0
+
+## Overview
+An image bucket app.
+
+## Architecture
+This app is written using Python 3.6, Django, and Postgres.
+
```

Comment:
You should add a section to this explaining how to get setup to run your app, like what environmental variables to export.